helm-chart/renku/values.yaml

```diff
@@ -1528,6 +1528,11 @@ dataService:
       pullPolicy: IfNotPresent
     total:
       resources: {}
+  k8sWatcher:
+    image:
+      repository: renku/data-service-k8s-watcher
+      tag: "0.37.1"
+      pullPolicy: IfNotPresent
```

Comment:
reminder: to submit PR to set resource requests in the gitops/devops repo so that deployments have limits/requests set.